features/admin.branding.v1/constants/branding-preferences-constants.ts

```diff
@@ -845,4 +845,11 @@ export class BrandingPreferencesConstants {
             }
         }
     };
+
+    public static readonly APP_WISE_BRANDING_FEATURE_TAG: string = "branding.applicationLevelBranding";
```

Comment:
You can use this constant for the above comment

Reply:
Fixed https://github.com/wso2/identity-apps/pull/6409/commits/1f0ef2552c269b4a70a36b8a12bb542f60fc1e78